scripts/list-repos-with-file-extensions.js

```diff
@@ -0,0 +1,49 @@
+const { listFileCountInRepository, searchByCriteria } = require("../util/github");
+
+module.exports = {
+    description: "list repos matching organization and extensions",
+    options: {
+        organization: {
+            alias: "o",
+            demandOption: true,
+            describe: "the GitHub organization to search",
+            type: "string",
+        },
+        extensions: {
+            alias: "e",
+            demandOption: true,
+            describe: "an array of extensions being searched for",
+            type: "array",
+        },
+        count: {
+            alias: "c",
+            default: false,
+            describe: "boolean flag to enable total count of files, will drastically increase execution time",
```

Comment:
we can reword this now, since this no longer impacts script execution time, right?

a small nitpick: the cli framework will tell users its a boolean, we don't need to include that in the description